src/CampaignKit.WorldMap/_references.js

```diff
@@ -1,25 +0,0 @@
-﻿// Copyright 2017-2019 Jochen Linnemann, Cory Gill
```

Comment:
Don't we need this anymore for JS Intellisense?

Reply:
You are correct.  I thought it was superseded with built in VS 2019 intellisense but it's not.  I added it back in.